### ISSUE TITLE
scavenger.js es7.array.includes not recognize with latest core-js

### ISSUE THREAD

**Issue by AmaelN** [wishtack/wishtack-steroids#189]
Hi,

I found your package really helpful, Thanks for sharing them with the community.

I'm having an issue with rx-scavenger.

I'm using angular 8.1 and I'm having the following error when I try to serve it :

```
ERROR in ./node_modules/@wishtack/rx-scavenger/dist/scavenger.js
Module not found: Error: Can't resolve 'core-js/es6/map' in 'C:\dev\node_modules\@wishtack\rx-scavenger\dist'
ERROR in ./node_modules/@wishtack/rx-scavenger/dist/scavenger.js
Module not found: Error: Can't resolve 'core-js/modules/es7.array.includes' in 'C:\dev\node_modules\@wishtack\rx-scavenger\dist'

```

Any idea how to reolve this issue ?

Thanks for your help in advance

**Comment by AmaelN** [comment]
This package is using "core-js": "^2.6.4",
while angular 8 is using "core-js": "^3.0.0",

Please let me know if there is a solutions for this

**Comment by AmaelN** [comment]
Alright,

I've updated savenger.js and package.js with the following :

Object.defineProperty(exports, "__esModule", { value: true });
require("core-js/es/map");
require("core-js/modules/es.array.includes");

and dependencies version

"core-js": "^3.3.2",

Now angular is not complaining.

**Comment by yjaaidi** [comment]
Hi @AmaelN, I am sorry for the inconvenience.
This has been solved by #190
Thank you for your feedback
